### ISSUE TITLE
Mobil nézet - Sponsors almenü

### ISSUE THREAD

**Issue by zsofimajor** [Drupalaton/drupalaton_profile#51]
Mobil nézetben a hamburger menüben a Sponsors almenüje bekavar, ezért nem lehet a Sponsors oldalra navigálódni. Be kéne rakni a Sponsorst is saját maga alá almenü pontnak?

**Comment by segi** [comment]
Igen, a jelenlegi megoldas nem jo mert hover esemeny nem letezik a telefonon(ami ugye lenyitna), egyedul a tap esemeny mukodik. Szoval egyelore nekem is ez az otletem.

**Comment by zsofimajor** [comment]
Legyen akkor így. Megnézed @segi pls?
